client/src/App.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
-import "./App.css";
-import Button from "./stories/Button";
+import GlobalStyle from "./GlobalStyle";
+import SplashPage from "./pages/SplashPage";
 
 function App() {
   return (
     <div className="App">
```

Comment:
```suggestion
    <div>
```
I think you can delete the className= "App" since you're also not using the App. css ;)

Reply:
thank you! 🤗